org.gnome.OfficeRunner.json

```diff
@@ -34,7 +34,11 @@
                 {
                     "type": "git",
                     "url": "https://gitlab.gnome.org/GNOME/office-runner.git",
-                    "commit": "15193aa95b2707619e4b54f7ff6b1f95b721f477"
+                    "commit": "ad339dde48d69bbaec76e87eec75cc37941fdad9"
```

Comment:
Why is the commit changed?

Reply:
To add some trivial changes like translation. I should be careful to stick on the stable releases.